targets/mega65/mega65.c

```diff
@@ -685,8 +697,17 @@ static void emulation_loop ( void )
 #ifdef		HAS_UARTMON_SUPPORT
 		if (XEMU_UNLIKELY(breakpoint_pc == cpu65.pc)) {
 			DEBUGPRINT("TRACE: Breakpoint @ $%04X hit, Xemu moves to trace mode after the execution of this opcode." NL, cpu65.pc);
+			m65mon_show_regs();
```

Comment:
gardners made a change in the hardware uartmon so that whenever a breakpoint or watchpoint hits, it spits out the register information.

Not only that, he injects an extra "comma" in this output, to help his m65 tooling detect when a breakpoint was hit.

I'll point out the location of that extra 'comma' in another comment...

So what you see here is me trying to replicate his new 'r' output, to enough of an extent to fool his 'm65' tool into thinking it's talking to real hardware.